corehq/apps/custom_data_fields/edit_model.py

```diff
@@ -104,7 +104,7 @@ def get_validator(cls, domain):
 
     @classmethod
     def page_name(cls):
-        return _("Edit {} Fields").format(cls.entity_string)
+        return _("Edit {} Fields").format(unicode(cls.entity_string))
```

Comment:
(won't block on it, but am curious to know the answer to the forcing to unicode question)